docs/tutorials/NEB/neb_tutorial.md

```diff
@@ -79,26 +79,26 @@ H  2.683149  3.028701  3.501215
 
 ### How many images do I need?
 
-It depends... It's usually easier to start with a small number of images and increase if you need to. 
+It depends... It's usually easier to start with a small number of images and increase if you need to. *[A bit vague.]*
 
-Using many (i.e. > 15) will result in slow convergence, but may lead to a more accurate minimum energy pathway. 
+Using many (i.e. > 15) will result in slow convergence, but may lead to a more accurate minimum energy pathway. *[Presumably more images will never give a worse result? Would it be better expressed in terms of a rule of thumb e.g. 10 x the number of expected transition states?]*
```

Comment:
I think it's quite tricky to give a rule of thumb for this, at least I don't feel qualified to do so.

Reply:
Fair enough - I think this addressed by the point above.

Reply:
```suggestion
Using many (i.e. > 15) will result in slow convergence, but may lead to a more accurate minimum energy pathway for complex barriers. If you do find a particularly complex pathway, however, you might be better off splitting the work up into more than one NEB calculation, each with fewer images. Just remember to always fully relax the end points (even if the end point is a meta-stable local minimum)!  
```